Palindrome.java

```diff
@@ -0,0 +1,70 @@
+
+import java.util.Scanner;
+
+public class Palindrome {
+	public static void main(String[] args) {
+		Scanner input = new Scanner(System.in);
+		int inputNumber;
+		int upperLimit = 1000000000;
+		int arraySize = 0;
+		int[] numbersArray;
+		boolean numberValid = true;
+		
+		System.out.print("Enter your number: ");
+		inputNumber = input.nextInt();
+		while (inputNumber < 0 || inputNumber > upperLimit) {
+			System.out.println("You can only enter value from 0 to " + upperLimit);
+			inputNumber = input.nextInt();
+		}
+		
+		for (int divisor = upperLimit; divisor > 1; divisor /= 10) {
+			int currentNumber = (int) ((inputNumber % divisor) / (divisor / 10));
```

Comment:
this works only because the upper limit is of the for 10^k.
I mean, if the upper limit is 550, this line will crash the application